op-node/sources/l1_client.go

```diff
@@ -24,6 +24,7 @@ type L1ClientConfig struct {
 func L1ClientDefaultConfig(config *rollup.Config, trustRPC bool, kind RPCProviderKind) *L1ClientConfig {
 	// Cache 3/2 worth of sequencing window of receipts and txs
 	span := int(config.SeqWindowSize) * 3 / 2
+	fullSpan := span
 	if span > 1000 { // sanity cap. If a large sequencing window is configured, do not make the cache too large
```

Comment:
what is the memory usage for the other components - this might be worth expanding for everything or limiting the span to be much larger.

Reply:
Opened CLI-3680 to track, I think it's a separate thing that we can improve, but it is not as urgent.